lib/wanda_web/schemas/v1/checks_customizations/custom_value.ex

```diff
@@ -0,0 +1,31 @@
+defmodule WandaWeb.Schemas.V1.ChecksCustomizations.CustomValue do
+  @moduledoc """
+  Custom value to be applied or already applied to a check
+  """
+
+  alias OpenApiSpex.Schema
+
+  require OpenApiSpex
+
+  OpenApiSpex.schema(
+    %{
+      title: "CustomValue",
+      description: "A single check from the catalog",
```

Comment:
I think we need to change the description

Reply:
here we go 63c8aa2d719cf27e0c61c9167df86c7058b7ecca

Reply:
I don't see the change :see_no_evil:

Reply:
Sorry did not push 😅  2e6b4850e991f53fd0a3eebc8418a4318cefee16